ext/moderation.py

```diff
@@ -0,0 +1,224 @@
+import disnake
+from disnake.ext import commands
+
+from utils.bot import Cog
+from utils.converter import TimeConverter as convert
+
+
+class moderation(Cog):
+    @commands.slash_command(name="ban")
+    @commands.has_permissions(ban_members=True)
+    async def ban(
+        self, inter: disnake.ApplicationCommandInteraction, member: disnake.Member, reason: str = "No reason"
+    ):
+        """Bans a member
+
+        Parameters
+        -----------
+        member: The member to ban
+        reason: The reason for banning
+        """
+        if inter.user.top_role > member.top_role:
+            try:
+                await inter.send(
+                    embed=disnake.Embed(
+                        title="Member banned",
+                        description=f"⚒ | {member.mention} has been banned from this server by {inter.user.mention} for **{reason}**!",
+                        color=disnake.Color.red(),
+                    )
+                )
+                await member.send(
+                    embed=disnake.Embed(
+                        title="Banned",
+                        description=f"You have been banned from {inter.guild.name} for **{reason}**!",
+                    )
+                )
```

Comment:
Only this should be in try block

Reply:
if i like dont add other things on the try block then without even member get banning from the server, the response will still get sent

Reply:
You can just use `finally`